runtime/runtime/src/metrics.rs

```diff
@@ -339,6 +343,60 @@ pub static CHUNK_RECORDED_SIZE_UPPER_BOUND_RATIO: Lazy<Histogram> = Lazy::new(||
     .unwrap()
 });
 
+static RECEIPT_FORWARDING_UNUSED_CAPACITY_GAS: Lazy<IntGaugeVec> = Lazy::new(|| {
+    try_create_int_gauge_vec(
+        "near_receipt_forwarding_unused_capacity_gas",
```

Comment:
nit: I'd be tempted to add "congestion" prefix to all the variable and metric names. (after the `near_` prefix that is)

Reply:
nit: I find it easier to work with non-negated values. Here it would be "used" instead of "unused". 

nit: We may also consider either also exposing the total capacity or making this a relative measure. It would be easier for the uninitiated to work with those metrics then.

Reply:
For the prefix, sure, I'll add them.

But the suggestion to have used instead of unused is not a nit, it's a discussion to show a different metric instead. I am happy you bring this up, this is a meaningful discussion to be had.

My thinking: Unused is the most meaningful value to display as a (sender,receiver) pair to understand congestion. All other congestion values are easier to look at at a single shard.

- How congested is shard X? Look at congestion_level(X).
- Is there backpressure? Look at all unused capacity and see if any is at zero. Values close to zero mean we almost have backpressure.

But when would you ever look at how much gas was forwarded? I can only imaging doing so to get a general understanding of cross-shard traffic flows. That's perhaps an interesting metric in general. But in terms of congestion, I learn nothing more than looking at the congestion level of the receiver.

Worse, to make sense if it, I have to compare it to total capacity. But if I have total capacity as a metric with (sender,receiver) pairs as index, people will create grafana boards where these N^2 numbers are shown. Something like 30 numbers that fluctuate wildly between 300'000 and 1000 Tgas by design, even when no congestion in happening. It will be a mess.

Instead, people should look at the congestion level of N shards (6 numbers). And if they must understand a specific shard better, look at unspent capacity. A number close to 0 is always a warning, a value at exactly 0 always means backpressure. Much easier to interpret.

Please let me know @wacban if you agree with my thinking or if you still think I should replace "unused capacity" with something else.

Reply:
I'm totally fine with this as is, it's only my personal preference. My thinking is that the pair of metrics (used, total) has full information and is simpler while we can still derive unused from it as well as other potentially interesting values like relative used or relative unused.

Reply:
I see. I agree, used + total could be stored and we can still display unused as the meaningful value.

The thing is, I think making it more general here is a trap. If we want to display forwarding between shards, it should really be a histogram, not a gauge. The more I think about it, the more I want to add it, in a separate PR. But this metric here is for a specific purpose (detecting backpressure) and generalizing it only deflects from that point and will lead to abuse of the data and confusion.